### PR TITLE
using-spotcollections.md updated

### PR DESCRIPTION
**Pull request by jesusgonzalez12**
Updated pyc8 code including fixes to be able to create a spot region and then to create a spot collection correctly.
Only the python code was changed, as part of the audit.
For more details, you can check the Kanban board.

### CODE REVIEW

File: docs/collections/documents/tutorials/using-spotcollections.md [hunk 1]
```diff
@@ -77,71 +77,148 @@ Let's assume your
...
+    {"_key": "1", "firstname": "Peter", "lastname": "Parker", "City": "NewYork"},
+    {"_key": "2", "firstname": "Bruce", "lastname": "Wayne", "City": "Gotham"},
+    {"_key": "3", "firstname": "Clark", "lastname": "Kent", "City": "Manhatten"},
+    {"_key": "4", "firstname": "Ned", "lastname": "Stark", "City": "Winterfell"},
+    {"_key": "5", "firstname": "Tywin", "lastname": "Lannister", "City": "Kings Landing"}
+]
+pp = pprint.PrettyPrinter(indent=4)
+
+
+# Step1: Open connection to GDN. You will be routed to closest region.
```
Comment (diana-macrometa) [inline docs/collections/documents/tutorials/using-spotcollections.md]: Put a space between the step and the number
Step 1
Step 2
etc.

File: docs/collections/documents/tutorials/using-spotcollections.md [hunk 2]
```diff
@@ -77,71 +77,148 @@ Let's assume your
...
+
+client.set_database_access_level(API_KEY_ID, spot_gf_name, "rw")
+
+#print(fabric.properties())
+client = spot_region.useFabric(spot_gf_name)
+#print(client.properties())
+
+spot_region_status = client.properties().get("options", {}).get("spotDc")
+if spot_region_status == "":
+    print("The fabric does not have a spot region registered")
```
Comment (diana-macrometa) [inline docs/collections/documents/tutorials/using-spotcollections.md]: Please end sentences with periods.

File: docs/collections/documents/tutorials/using-spotcollections.md [hunk 3]
```diff
@@ -77,71 +77,148 @@ Let's assume your
...
-      clientx = C8Client(protocol='https', host=region_url, port=443)
-      tenantx = clientx.tenant(email, password)
-      fabricx = tenantx.useFabric(geo_fabric)
-      cursorx = fabricx.c8ql.execute(read_query)
-      docs = [document for document in cursorx]
-      pp.pprint(docs)
-
-    # Step5: Delete Data
-    print("5. DELETE_DATA: region: {}, collection: {}".format(global_url, collection_name))
+        print(collection_name, "is NOT a spot collection")
```
Comment (diana-macrometa) [inline docs/collections/documents/tutorials/using-spotcollections.md]:
```suggestion
        print(collection_name, "is not a spot collection.")
```

File: docs/collections/documents/tutorials/using-spotcollections.md [hunk 4]
```diff
@@ -77,71 +77,148 @@ Let's assume your
...
+    print("The NEW query has been saved")
+else:
+    print("Query already exists")
+
+
+print(client.execute_restql(read_query_name))
+#print(client.execute_query("FOR account IN accounts RETURN account"))
+
+
+# Step5: Read Data from other regions.
```
Comment (diana-macrometa) [inline docs/collections/documents/tutorials/using-spotcollections.md]:
```suggestion
# Step 5: Read data from other regions.
```

File: docs/collections/documents/tutorials/using-spotcollections.md [hunk 5]
```diff
@@ -77,71 +77,148 @@ Let's assume your
...
+for region_url in region_urls:
+    print(f"\n5. DATA_MOBILITY: Reading from region: {region_url}")
+    #clientx = C8Client(protocol='https', host=region_url, port=443, geofabric=
+        #spot_gf_name, apikey= API_KEY)
+    print(client.execute_restql(read_query_name))
+
+
+# Step6: Delete Data
```
Comment (diana-macrometa) [inline docs/collections/documents/tutorials/using-spotcollections.md]:
```suggestion
# Step 6: Delete data.
```

**Review by diana-macrometa — CHANGES_REQUESTED** [review]

File: docs/collections/documents/tutorials/using-spotcollections.md (hunk 2 above)
Comment (jesusgonzalez12) [reply docs/collections/documents/tutorials/using-spotcollections.md]: thanks, done.

File: docs/collections/documents/tutorials/using-spotcollections.md (hunk 3 above)
Comment (jesusgonzalez12) [reply docs/collections/documents/tutorials/using-spotcollections.md]: done :)

File: docs/collections/documents/tutorials/using-spotcollections.md (hunk 1 above)
Comment (jesusgonzalez12) [reply docs/collections/documents/tutorials/using-spotcollections.md]: Thanks, Diana. Done :)

File: docs/collections/documents/tutorials/using-spotcollections.md (hunk 4 above)
Comment (jesusgonzalez12) [reply docs/collections/documents/tutorials/using-spotcollections.md]: done :)

File: docs/collections/documents/tutorials/using-spotcollections.md (hunk 5 above)
Comment (jesusgonzalez12) [reply docs/collections/documents/tutorials/using-spotcollections.md]: done :)

**Comment by jesusgonzalez12** [comment]
Updated with suggestions.
Thanks for reviewing this.

**Review by jamesm-macrometa — APPROVED** [review]

**Comment by jamesm-macrometa** [comment]
@diana-macrometa Just FYI this PR is blocked from merging and needs your approval 😉

File: docs/collections/documents/tutorials/using-spotcollections.md [hunk 6]
```diff
@@ -77,71 +77,148 @@ Let's assume your
...
+
+# Step 3: Insert data into collection.
+#get_keys = client.execute_restql("accounts_keys")
+print(f"3. INSERT_DATA: region: {URL}, collection: {collection_name}")
+#collection.insert_many(data)
+print("Data has been inserted successfully inserted (duplicating data was avoided).")
+
+
+
+# Step 4: Create a new query worker or look for the existing one, and read data from the collection.
```
Comment (diana-macrometa) [inline docs/collections/documents/tutorials/using-spotcollections.md]:
```suggestion
# Step 4: Create a new query worker or look for the existing one, then read data from the collection.
```

**Review by diana-macrometa — APPROVED** [review]
Once super minor suggestion. Great work!

File: docs/collections/documents/tutorials/using-spotcollections.md (hunk 6 above)
Comment (jesusgonzalez12) [reply docs/collections/documents/tutorials/using-spotcollections.md]: Sure, thanks @diana-macrometa

**Comment by diana-macrometa** [comment]
@jesusgonzalez12, please merge this at your convenience.

**Comment by jesusgonzalez12** [comment]
Thanks @diana-macrometa. The PR has been merged.

**Pull request merged**